src/core/action/noise/generator/RandomNumberGeneratorService.java

```diff
@@ -0,0 +1,22 @@
+package core.action.noise.generator;
+
+import java.util.Random;
+
+public class RandomNumberGeneratorService {
```

Comment:
To tengo en el codigo en que estuve trabajando, una clase con CASI el mismo nombre. Vamos a meterlo todo junto, claramente, una vez que lo suba

Reply:
Bueno, seria un metodo mas que le agreguemos